vtysh/extract.pl.in

```diff
@@ -143,6 +143,9 @@ sub scan_file {
         elsif ($fabricd) {
            $protocol = "VTYSH_FABRICD";
         }
+	elsif ($file =~ /pimd\/pim6_cmd\.c$/) {
```

Comment:
Since this file is not added in this PR, can you please move these 3 lines to PR #10387 where the file is actually added?

Reply:
I will just comment this line, since the below line can't be moved into PR https://github.com/FRRouting/frr/pull/10387, it will throw compilation error.

Reply:
Compilation error? In the perl script? I'm confused

Reply:
Aah my bad :) 
Sure I will take care this as part of https://github.com/FRRouting/frr/pull/10393